oclc.py

```diff
@@ -1333,7 +1347,10 @@ def process_item(self, identifier):
             else:
                 exception = "OCLC raised an error: %r" % e
                 transient = True
-            return CoverageFailure(identifier, exception, transient=transient)
+            return CoverageFailure(
+                identifier, exception, data_source=self.output_source,
```

Comment:
The absence of `data_source` here sometimes meant that `CoverageFailure` objects couldn't be converted to `CoverageRecord` objects.